### PR TITLE
File Reconstruction Dashboard

### PR DESCRIPTION
**Pull request by acarb95**
This contains the file reconstruction dashboard, I haven't been able to build it yet. I created it on a QA machine (244), so it should work, assuming I didn't make a typo deleting the stuff up to source.

Right now it contains the "Attach" column in the table. That displays a True or False value. When Alex gets the downloading done, it will display a checkbox where they user can download the attachment. 

If someone could pull it down and test it, that would be great. Already got it passed by Rob.

### CODE REVIEW

**Comment by craig-cogdill** [comment]
This PR has the same issue that Shweta had. When these json files are inserted into ES, their ElasticSearch ID is autogenerated as their filename. Your dashboard is looking for ES ID's that don't exist since your filenames are different from what they were when you built the dashboard.

File: resources/dashboards/File-Reconstruction-Dashboard.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "File Reconstruction Dashboard",
+  "hits": 0,
+  "description": "",
+  "panelsJSON": "[{\"id\":\"Top-10-Attach-Types-(bar-graph)\",\"type\":\"visualization\",\"size_x\":6,\"size_y\":4,\"col\":1,\"row\":1},{\"id\":\"Top-10-Attachment-Names\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":1},{\"id\":\"Top-10-Attachment-Types-by-Total-Bytes-Delta\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":3},{\"id\":\"Top-10-Email-Receivers-by-Attachment-Type\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":3},{\"id\":\"Top-10-Senders-by-Attachment-Type\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":1},{\"id\":\"Attachment-Search\",\"type\":\"search\",\"size_x\":12,\"size_y\":4,\"col\":1,\"row\":5,\"columns\":[\"SenderEmail\",\"ReceiverEmail\",\"AttachFilenameCdispo\",\"AttachSize\",\"AttachType\",\"Application\"],\"sort\":[\"TimeUpdated\",\"desc\"]}]",
```
Comment (craig-cogdill) [inline resources/dashboards/File-Reconstruction-Dashboard.json]: You have to update the dashboard (to use the new filenames) either by recreating and exporting it through the UI, or change the panel ID's here to reflect their new file names

**Comment by craig-cogdill** [comment]
Can you make a card in Rally to add the file reconstruction to the Table on this dashboard after Alex has finished working on that?

**Comment by acarb95** [comment]
@craig-cogdill No need to add the file reconstruction to the table, it's kinda already there. He said that the file reconstruction should be the Attach column in the table. So by having the attached column, when his changes get merged, that will change from True/False to check boxes (from my understanding)

**Comment by craig-cogdill** [comment]
I don't see the ```Attach``` column in your table now that I look at it though

**Comment by craig-cogdill** [comment]
Comments:
- Sender / Receiver email charts should be by Count:SenderEmail.raw (with query Attach:true) instead of Attachment type
- Re-add the search with Attach column to the dashboard and re-export the dashboard
- Waiting on decision for Attachment Names

**Comment by craig-cogdill** [comment]
New Comments:
- Put Attach to the left so that you can download without scrolling
- Rename the AttachType viz so that it is locatable.
- Repull the Table search into the dashboard so that AttachFilenameCDispo becomes Filename

**Comment by KjellKod** [comment]
:+1:

**Comment by KjellKod** [comment]
:+1: from Rob

**Pull request merged**